### ISSUE TITLE
children router bug

### ISSUE THREAD

**Issue by allmors** [ItalyPaleAle/svelte-spa-router#323]
The type of effect I want is vue-router routing with nested sub-routing. Here is my example. My different paths correspond to different components, but the plug-in seems to be that if this path is used by other routes, the same type cannot be used.
![image](https://github.com/ItalyPaleAle/svelte-spa-router/assets/93471589/3805f8dc-8b35-4b55-89ac-275868f35d90)
![image](https://github.com/ItalyPaleAle/svelte-spa-router/assets/93471589/1f06a8f9-eae4-41a8-9b6e-a3af6c2b7ae9)
![image](https://github.com/ItalyPaleAle/svelte-spa-router/assets/93471589/95828853-89e6-4428-8b62-97dcbbfcdb94)

Unable to achieve effect
```
const routes = {
    '/:blacklist': BlackList,
    '/:setting': Setting,
}
```

can achieve the effect
```
const routes = {
    '/:blacklist': BlackList,
    '/:setting/:xxx': Setting,
}
```

But what I want is that different components can be used with different pathnames in the relevant cascade

In short, it is as follows：
```
const routes = {
    '/:blacklist': BlackList,
    '/:setting': Setting,
}
```

- `/dashboard/blacklist` Use BlackList component
- `/dashboard/setting` Use Setting component

**Comment by ItalyPaleAle** [comment]
Do you set the `prefix` to the sub-routers? See docs/examples: https://github.com/ItalyPaleAle/svelte-spa-router/blob/main/Advanced%20Usage.md#nested-routers
